chalice/config.py

```diff
@@ -74,13 +74,16 @@ class Config(object):
     """
 
     def __init__(self,
-                 chalice_stage=DEFAULT_STAGE_NAME,
-                 function_name=DEFAULT_HANDLER_NAME,
-                 user_provided_params=None,
-                 config_from_disk=None,
-                 default_params=None,
-                 layers=None):
-        # type: (str, str, StrMap, StrMap, StrMap, List[str]) -> None
+                 chalice_stage=DEFAULT_STAGE_NAME,      # type: str
+                 function_name=DEFAULT_HANDLER_NAME,    # type: str
+                 user_provided_params=None,             # type: StrMap
+                 config_from_disk=None,                 # type: StrMap
+                 default_params=None,                   # type: StrMap
+                 layers=None,                           # type: List[str]
+                 rest_api_domain_name=None,             # type: StrMap
```

Comment:
I don't think we need to pass these in (not sure why we're passing in layers directly but we shouldn't need that either).